src/app/pages/TokenDashboardPage/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useScreenSize } from '../../hooks/useScreensize'
+import { FC } from 'react'
+import { PageLayout } from '../../components/PageLayout'
+import Divider from '@mui/material/Divider'
+import { TokenTitleCard } from './TokenTitleCard'
+import { TokenSnapshot } from './TokenSnapshot'
+import { TokenDetailsCard } from './TokenDetailsCard'
+import { useRequiredScopeParam } from '../../hooks/useScopeParam'
+import { useLoaderData } from 'react-router-dom'
+import { useTokenInfo } from './hook'
+import { AppErrors } from '../../../types/errors'
+
+export const TokenDashboardPage: FC = () => {
+  const { isMobile } = useScreenSize()
+  const scope = useRequiredScopeParam()
+  const address = useLoaderData() as string
+
+  const { isError } = useTokenInfo(scope, address)
```

Comment:
Wouldn't this be better if we pass the response of this hook into child components, instead of reusing the same hook inside `TokenSnapshot` for example. I know that response is cached.

Reply:
I dunno, I was trying to follow the patterns on the old Dashboard. (Now called ParatimeDashboard.) There also, multiple cards are working with basically the same data (runtime status and stats), but the only sharing is happening on the query level.